dom/telephony/gonk/TelephonyService.js

```diff
@@ -2127,42 +2150,25 @@ TelephonyService.prototype = {
   },
 
   sendUSSD: function(aClientId, aUssd, aCallback) {
+    aCallback.notifyDialMMI(MMI_KS_SC_USSD);
```

Comment:
This is needed to "prepare" the callback for notifyDialMMISuccess or notifyDialMMIError